schema/openapi.json

```diff
@@ -0,0 +1,6193 @@
+{
+    "openapi": "3.1.0",
+    "info": {
+        "title": "Infrahub",
+        "version": "1.5.0b1"
```

Comment:
I hadn't considered the version here. It feels like something that will pop up as merge conflicts when we merge between `stable` and `develop`. Not really a huge issue but something to consider. I'm not sure we strictly need the version here in order to do bindings for the frontend. Might be that we should remove it before saving the file and just have a blank string, or "latest" as the version. Do you have any thoughts around this?